.vscode/settings.json

```diff
@@ -1,5 +1,10 @@
 {
```

Comment:
We probably need to add .vscode to ignore, so that we can each use our individual settings.

Reply:
Your changes are the same as the settings on master, no conflicts or changes have been put into effect in this PR.

The commit you are seeing isn't because you changed anything it is because you merge some changes into your branch instead of rebasing. The files changed looks a bit cleaner if you rebase main instead of merging main in.

Reply:
recommendation from stackoverflow is to keep settings json https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control